packages/edit-site/src/components/editor/index.js

```diff
@@ -39,15 +39,33 @@ import { SidebarComplementaryAreaFills } from '../sidebar';
 import BlockEditor from '../block-editor';
 import KeyboardShortcuts from '../keyboard-shortcuts';
 import GlobalStylesProvider from './global-styles-provider';
+import NavigationSidebar from '../navigation-sidebar';
 
 const interfaceLabels = {
 	leftSidebar: __( 'Block Library' ),
 };
 
 function Editor() {
-	const [ isInserterOpen, setIsInserterOpen ] = useState( false );
+	const [ isInserterOpen, _setIsInserterOpen ] = useState( false );
```

Comment:
Provided that we want this behavior, it might make sense to collapse these states into one `leftSidebarContent` that can have on of three possible values for now: `null`, `navigation`, and `inserter`. With present setup adding another element that wants to toggle into this space (there were some plans for doing that with block navigation) will be cumbersome, since each element will have to be aware of others and the toggling logic will have to be updated everywhere.

It would also remove the need for current naming with leading underscore (`setIsInserterOpen` and `_setIsInserterOpen`), which I find hard to read.

Reply:
Done!